docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -0,0 +1,84 @@
+---
+assignees:
+- davidopp
+
+---
+ 
+{% capture overview %}
+This page shows how to safely drain a machine, respecting the application-level
+disruption SLOs you have specified using PodDisruptionBudget.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* We assume you are using Kubernetes release >= 1.5.
+
+* We assume you have created [PodDisruptionBudget(s)](../../admin/disruptions.md) to express the
+application-level disruption SLOs you want the system to enforce.
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Use `kubectl drain` to remove a node from service
+
+You can use `kubectl drain` to safely evict all of your pods from a node before you perform maintenance
+on the node (e.g. kernel upgrade, hardware maintenance, etc.). To define a few terms:
```

Comment:
Recommend a slight restructure. Instead of "to define a few terms" and a bullet list, structure it like this:

"You can use `kubectl drain` to safely evict all of your pods from a node before you perform maintenance on the node (e.g. kernel upgrade, hardware maintenance, etc.). Safe evictions allow the pod's containers to [gracefully terminate](/docs/user-guide/production-pods.md#lifecycle-hooks-and-termination-notice) and will respect the `PodDisruptionBudgets` you have specified.

**Note:** `kubectl drain` will ignore certain system pods on the node that cannot be killed; see the [kubectl drain](/docs/user-guide/kubectl/kubectl_drain.md) documentation for more details."